### PR TITLE
[86c0f12gm][dropdown-menu] added new DropdownMenu.Item.Text for wrapping text if used with addons

### PR DESCRIPTION
**Pull request by ilyabrower**
## Motivation and Context
Previously, when using `Addons` in `DropdownMenu.Item`, there was no margin between them and the text. I added a new component to wrap the text in this case, it adds the necessary margin.

## How has this been tested?
Manually

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [X] Nice improve.

## Checklist:

- [X] My code follows the code style of this project.
- [X] I have updated the documentation accordingly or it's not required.
- [X] Unit tests are not broken.
- [X] I have added changelog note to corresponding `CHANGELOG.md` file with planned publish date.
- [ ] I have added new unit tests on added of fixed functionality.

### CODE REVIEW

File: semcore/dropdown-menu/src/style/dropdown-menu.shadow.css
Comment (sheila-semrush) [inline semcore/dropdown-menu/src/style/dropdown-menu.shadow.css:1]: Looks good!
I replaced the icon with a more neutral one, so nobody gets any wrong ideas :)
Do you think we could align the icon vertically by default more neatly with the text? Right now it's too high.
image

Comment (ilyabrower) [reply semcore/dropdown-menu/src/style/dropdown-menu.shadow.css:1]: yes, we could, added.

Comment (sheila-semrush) [reply semcore/dropdown-menu/src/style/dropdown-menu.shadow.css:1]: thanks, looks great!

Comment (sheila-semrush) [reply semcore/dropdown-menu/src/style/dropdown-menu.shadow.css:1]: I hope they don't add multiple lines of text into Item.Content 😅

**Review by sheila-semrush — APPROVED** [review]
everything's great from my side

File: semcore/dropdown-menu/src/style/dropdown-menu.shadow.css
Comment (ilyabrower) [reply semcore/dropdown-menu/src/style/dropdown-menu.shadow.css:1]: Me too =)

**Review by Valeria-Zimnitskaya — APPROVED** [review]

**Pull request merged**